src/api/plugins/video/warren_video/models.py

```diff
@@ -1 +1,26 @@
 """Defines the schema of what is returned inside the API responses."""
+
+from pydantic import BaseModel, validator
+
+Seconds = int | None
+Percent = float | None
+
+
+class Info(BaseModel):
+    name: str | None
+    length: Seconds
+    completion_threshold: Percent
```

Comment:
Why not using simple types?

```suggestion
    length: int
    completion_threshold: float
```

Also: are we sure we want all fields to be nullable?

Reply:
In terms of readability, having to guess the unit of a variable seems pretty annoying for future devs. I am totally open to discuss it. Type alias also offered me to factorize the Union of the nullable and simple types, avoiding repeating it.

Reply:
I am now using simple types, and added an explicit comment. WDYT?